src/components/editor/ResourceTemplate.jsx

```diff
@@ -1,50 +1,31 @@
-// Copyright 2018 Stanford University see LICENSE for license
+// Copyright 2019 Stanford University see LICENSE for license
 
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import ResourceTemplateForm from './ResourceTemplateForm'
-import { rootResourceTemplateLoaded } from 'actions/index'
-import { getResourceTemplate } from 'sinopiaServer'
+import { fetchRootResourceTemplate } from 'actionCreators'
 import { rootResource } from 'selectors/resourceSelectors'
 
-const _ = require('lodash')
-
 /**
  * This is the root component of the editor on the resource edit page
  */
 class ResourceTemplate extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {}
-  }
-
   // Called immediately after the component is rendered for the first time
   componentDidMount() {
-    if (!this.props.resourceTemplate) {
-      this.resolveResourceTemplatePromise(getResourceTemplate(this.props.resourceTemplateId))
-    }
-  }
-
-  resolveResourceTemplatePromise = (promise) => {
-    promise.then((responseAndBody) => {
-      this.props.handleResourceTemplate(responseAndBody.response.body)
-    }).catch((error) => {
-      console.error(error)
-      this.setState({ error })
-    })
+    this.props.retrieveResourceTemplate(this.props.resourceTemplateId)
   }
 
   render() {
-    if (_.isEmpty(this.props.resourceTemplate)) {
-      let errorMessage = <span/>
+    let errorMessage = <span/>
 
-      if (this.state.error) {
-        errorMessage = <div className="alert alert-warning">Sinopia server is offline or has no resource templates to display</div>
-      }
-      return errorMessage
+    if (this.props.error) {
+      errorMessage = <div className="alert alert-warning">Sinopia server is offline or has no resource templates to display.</div>
```

Comment:
is this error message still correct?  Is this component loading the root template or any resource template?  (and if it's template singular, is should this error message be singular too?  Is it more like "Sinopia server is offline or unable to retrieve resource template (id of resource template)" ?